From 9943d8be7fb58784ece6cf8000b5da88743a533e Mon Sep 17 00:00:00 2001
From: Predators <31931530+predators46@users.noreply.github.com>
Date: Tue, 20 Dec 2022 08:09:15 +0800
Subject: [PATCH] Add files via upload

---
 mysys/thr_lock.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mysys/thr_lock.c b/mysys/thr_lock.c
index b1722e0..57568af 100644
--- a/mysys/thr_lock.c
+++ b/mysys/thr_lock.c
@@ -137,7 +137,7 @@ static inline pthread_cond_t *get_cond(void)
 */
 
 
-inline int LOCK_CMP(THR_LOCK_DATA *A, THR_LOCK_DATA *B)
+static inline int LOCK_CMP(THR_LOCK_DATA *A, THR_LOCK_DATA *B)
 {
   if (A->lock != B->lock)
     return A->lock < B->lock;
