From 0ebd38a585472e99cefa0641555b5250e3507315 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Fri, 12 Apr 2024 10:50:45 +0800
Subject: [PATCH] Update meson.build

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 8963b31..0792cf9 100644
--- a/meson.build
+++ b/meson.build
@@ -920,7 +920,7 @@ if get_option('allow-kcmp') \
   pre_args += '-DALLOW_KCMP'
 endif
 
-prog_python = import('python').find_installation('python3')
+prog_python = import('python').find_installation('$(STAGING_DIR_HOSTPKG)/bin/$(PYTHON3)')
 has_mako = run_command(
   prog_python, '-c',
   '''
