From 388d509ee6fd13fba69d93eac2a6dafe317c75bd Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Fri, 12 Apr 2024 18:44:38 +0800
Subject: [PATCH] Update meson.build

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 8963b31..5dd70cf 100644
--- a/meson.build
+++ b/meson.build
@@ -920,7 +920,7 @@ if get_option('allow-kcmp') \
   pre_args += '-DALLOW_KCMP'
 endif
 
-prog_python = import('python').find_installation('python3')
+prog_python = import('python').find_installation('/home/runner/work/mosdns/mosdns/sdk/staging_dir/hostpkg/bin/python3.11')
 has_mako = run_command(
   prog_python, '-c',
   '''
