From ec4cd28399f79d01b15112fe81f913859f3b5a08 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Fri, 12 Apr 2024 19:25:43 +0800
Subject: [PATCH] Update meson.build

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 8963b31..0af6a3f 100644
--- a/meson.build
+++ b/meson.build
@@ -920,7 +920,7 @@ if get_option('allow-kcmp') \
   pre_args += '-DALLOW_KCMP'
 endif
 
-prog_python = import('python').find_installation('python3')
+prog_python = import('python').find_installation('python3.11')
 has_mako = run_command(
   prog_python, '-c',
   '''
