From 56617a08adb7a401b5016cb878871e22c75a65a6 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 28 Aug 2024 10:01:37 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 3b64c2f..5dd3619 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -6,7 +6,7 @@
 build-backend = 'mesonpy'
 backend-path = ['.']
 requires = [
-  'meson >= 0.63.3',
+#  'meson >= 0.63.3',
   'pyproject-metadata >= 0.7.1',
   'tomli >= 1.0.0; python_version < "3.11"',
   'setuptools >= 60.0; python_version >= "3.12"',
@@ -29,7 +29,7 @@ classifiers = [
 
 dependencies = [
   'colorama; os_name == "nt"',
-  'meson >= 0.63.3',
+#  'meson >= 0.63.3',
   'pyproject-metadata >= 0.7.1',
   'tomli >= 1.0.0; python_version < "3.11"',
   'setuptools >= 60.0; python_version >= "3.12"',
