From 82376d107f55bca495f2caea68fc301db7511412 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 28 Aug 2024 01:57:09 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index af6814a..4f62b1e 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -3,11 +3,11 @@
 # SPDX-License-Identifier: MIT
 
 [build-system]
-build-backend = 'mesonpy'
+#build-backend = 'mesonpy'
 backend-path = ['.']
 requires = [
-  'meson >= 0.63.3; python_version < "3.12"',
-  'meson >= 1.2.3; python_version >= "3.12"',
+#  'meson >= 0.63.3; python_version < "3.12"',
+#  'meson >= 1.2.3; python_version >= "3.12"',
   'packaging >= 19.0',
   'pyproject-metadata >= 0.7.1',
   'tomli >= 1.0.0; python_version < "3.11"',
@@ -33,8 +33,8 @@ classifiers = [
 ]
 
 dependencies = [
-  'meson >= 0.63.3; python_version < "3.12"',
-  'meson >= 1.2.3; python_version >= "3.12"',
+#  'meson >= 0.63.3; python_version < "3.12"',
+#  'meson >= 1.2.3; python_version >= "3.12"',
   'packaging >= 19.0',
   'pyproject-metadata >= 0.7.1',
   'tomli >= 1.0.0; python_version < "3.11"',
