From a109dfd5084d448f92032615a7cfa8cb3f6df396 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Mon, 2 Sep 2024 21:26:05 +0800
Subject: [PATCH] Update configure.ac

---
 configure.ac | 99 +---------------------------------------------------
 1 file changed, 1 insertion(+), 98 deletions(-)

diff --git a/configure.ac b/configure.ac
index 9a5c252..66d8757 100644
--- a/configure.ac
+++ b/configure.ac
@@ -130,7 +130,7 @@ AC_ARG_WITH(perl-lib,
 	else
 		want_perl=yes
 		perl_prefix_note=no
-		PERL_MM_PARAMS="INSTALLDIRS=perl LIB=$withval"
+		PERL_MM_PARAMS="INSTALLDIRS=perl INSTALLPRIVLIB=$withval INSTALLARCHLIB=$withval INSTALLSITELIB=$withval INSTALLSITEARCH="
 		perl_set_use_lib=yes
 	fi,
 	want_perl=yes)
@@ -392,70 +392,6 @@ dnl **
 if test "$want_perl" != "no"; then
 	AC_MSG_CHECKING(for working Perl support)
 
-	if test -z "$perlpath"; then
-		perl_check_error="perl binary not found"
-	else
-		PERL_CFLAGS=`$perlpath -MExtUtils::Embed -e ccopts 2>/dev/null`
-	fi
-
-	if test "x$ac_cv_prog_gcc" = "xyes" -a -z "`echo $host_os|grep 'bsd\|linux'`"; then
-		dnl * several systems have Perl compiled with native compiler
-		dnl * but irssi is being compiled with GCC. Here we try to
-		dnl * fix those command line options a bit so GCC won't
-		dnl * complain about them. Normally there's only few options
-		dnl * that we want to keep:
-		dnl * -Ddefine -Uundef -I/path -fopt -mopt -iwithsysroot
-		PERL_CFLAGS=`echo $PERL_CFLAGS | $perlpath -pe 's/^(.* )?-@<:@^DUIifm@:>@@<:@^ @:>@+/\1/g; s/^(.* )?\+@<:@^ @:>@+/\1/g'`
-
-		PERL_EXTRA_OPTS="CCCDLFLAGS=\"-fPIC\""
-		AC_SUBST(PERL_EXTRA_OPTS)
-	fi
-
-	if test -z "$PERL_CFLAGS"; then
-		if test -n "$perl_check_error"; then
-			perl_check_error="Error getting perl CFLAGS"
-		fi
-		AC_MSG_RESULT([not found, building without Perl])
-		want_perl=no
-	else
-		PERL_LDFLAGS=`$perlpath -MExtUtils::Embed -e ldopts 2>/dev/null`
-
-		dnl * remove all database stuffs
-		dnl * nsl is already in ldflags
-		dnl * libc is of course linked without needing -lc
-		dnl * -rdynamic must not be in LIBADD line
-		for word in -ldb -ldbm -lndbm -lgdbm -lc -rdynamic; do
-		  PERL_LDFLAGS=`echo $PERL_LDFLAGS | $sedpath -e "s/$word //" -e "s/$word$//"`
-		done
-
-		case "$host_os" in
-		  linux*)
-		    PERL_LDFLAGS=`echo $PERL_LDFLAGS | $sedpath -e 's/-lposix //' -e 's/-lposix$//'`
-		    ;;
-		  hpux*)
-		    if test "x$ac_cv_prog_gcc" = "xyes"; then
-		      PERL_CFLAGS=`echo $PERL_CFLAGS | $sedpath -e 's/-Ae //' -e 's/-Ae$//'`
-		      PERL_LDFLAGS=`echo $PERL_LDFLAGS | $sedpath -e 's/-Ae //' -e 's/-Ae$//'`
-		    fi
-		    ;;
-		  *)
-		    ;;
-		esac
-
-		dnl * check that perl's ldflags actually work
-		echo "#include <EXTERN.h>" > conftest.c
-		echo "#include <perl.h>" >> conftest.c
-		echo "int main(){perl_alloc(); return 0;}" >> conftest.c
-		$CC $CFLAGS $PERL_CFLAGS conftest.c -o conftest $LDFLAGS $PERL_LDFLAGS 2> perl.error.tmp > /dev/null
-		if test ! -s conftest -a "x$ignore_perl_errors" = "x"; then
-			perl_check_error="Error linking with perl libraries: $PERL_LDFLAGS: `cat perl.error.tmp`"
-			AC_MSG_RESULT([error linking with perl libraries, building without Perl])
-			want_perl=no
-		fi
-
-		rm -f perl.error.tmp
-	fi
-
 	if test "x$want_perl" != "xno"; then
 		AC_MSG_RESULT(ok)
 
@@ -489,39 +425,6 @@ if test "$want_perl" != "no"; then
 			PERL_STATIC_LIBS=0
 		fi
 
-		# remove any prefix from PERL_MM_OPT
-		PERL_MM_OPT=`perl -MText::ParseWords -e 'sub qu{$_=shift;s{^(.*?)=(.*)$}{($a,$b)=($1,$2);$b=~s/"/\\\\"/g;qq{$a="$b"}}ge if /@<:@\s"@:>@/;$_} local $,=" "; print map qu($_), grep !/^(INSTALL_BASE|PREFIX)=/, shellwords(@ARGV)' "$PERL_MM_OPT"`
-		# figure out the correct @INC path - we'll need to do this
-		# through MakeMaker since it's difficult to get it right
-		# otherwise.
-		$perlpath -MExtUtils::MakeMaker -e 'WriteMakefile(NAME => "test", MAKEFILE => "Makefile.test", FIRST_MAKEFILE => "/dev/null", NO_META => 1, NO_MYMETA => 1);' $PERL_MM_PARAMS >/dev/null
-		echo 'show-INSTALLDIRS:' >> Makefile.test
-		echo '	@echo $(INSTALLDIRS)' >> Makefile.test
-		perl_INSTALLDIRS=`$am_make -s -f Makefile.test show-INSTALLDIRS`
-		if test "x$perl_INSTALLDIRS" = "xsite"; then
-			perl_library_dir="site default"
-			perl_INSTALL_VAR=INSTALLSITEARCH
-		elif test "x$perl_INSTALLDIRS" = "xvendor"; then
-			perl_library_dir="vendor default"
-			perl_INSTALL_VAR=INSTALLVENDORARCH
-		else
-			perl_library_dir="module default"
-			perl_INSTALL_VAR=INSTALLARCHLIB
-		fi
-		echo 'show-ARCHLIB:' >> Makefile.test
-		echo '	@echo $('"$perl_INSTALL_VAR"')' >> Makefile.test
-		perl_use_lib=`$am_make -s -f Makefile.test show-ARCHLIB`
-		rm -f Makefile.test
-		if test "x$perl_set_use_lib" = "xyes"; then
-			if $perlpath -e 'exit ! grep $_ eq $ARGV@<:@0@:>@, grep /^\//, @INC' "$perl_use_lib"; then
-				perl_library_dir="other path in @INC"
-				perl_set_use_lib=no
-			else
-				perl_library_dir="prepends to @INC with /set perl_use_lib"
-				PERL_USE_LIB="$perl_use_lib"
-			fi
-		fi
-
 		AC_SUBST(perl_module_lib)
 		AC_SUBST(perl_static_lib)
 		AC_SUBST(perl_module_fe_lib)
