From 19f07ae05282313478abbfc4e0cc1f3fc98614d4 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 28 Aug 2024 23:00:20 +0800
Subject: [PATCH] Update meson.build

---
 lib/contourpy/util/meson.build | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/lib/contourpy/util/meson.build b/lib/contourpy/util/meson.build
index 568ccbe..59b750b 100644
--- a/lib/contourpy/util/meson.build
+++ b/lib/contourpy/util/meson.build
@@ -63,10 +63,10 @@ conf_data.set('linker_id', cpp.get_linker_id())
 conf_data.set('compile_command', ' '.join(cpp.cmd_array()))
 
 # Host and build machines
-machines = {
-  'host': host_machine,
-  'build': build_machine,
-}
+#machines = {
+#  'host': host_machine,
+#  'build': build_machine,
+#}
 foreach name, machine : machines
   conf_data.set(name + '_cpu', machine.cpu())
   conf_data.set(name + '_cpu_family', machine.cpu_family())
