From 08d61e4b10e2a3ed0fe832d94ec3cff47d724c36 Mon Sep 17 00:00:00 2001
From: esaapriliasalsabila <esaapriliasalsabila@gmail.com>
Date: Tue, 23 Jul 2024 14:03:56 +0800
Subject: [PATCH] Fix use of obsolete macros

---
 configure.in | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/configure.in b/configure.in
index 94a9d5b..0486319 100644
--- a/configure.in
+++ b/configure.in
@@ -639,7 +639,7 @@ fi
 
 AC_MSG_CHECKING(whether features provided by the user community should be included.)
 AC_ARG_ENABLE(community-features,
-    AC_HELP_STRING(
+    AS_HELP_STRING(
         [--disable-community-features], 
         [Disable additional features provided by the user community.]),
     [ ENABLE_COMMUNITY_FEATURES=$enableval ],
@@ -1715,7 +1715,7 @@ fi
 
 # If we should allow error injection tests
 AC_ARG_WITH(error-inject,
-    AC_HELP_STRING([--with-error-inject],[Enable error injection in MySQL Server]),
+    AS_HELP_STRING([--with-error-inject],[Enable error injection in MySQL Server]),
     [ with_error_inject=$withval ],
     [ with_error_inject=no ])
 
@@ -1729,7 +1729,7 @@ then
 fi
 
 AC_ARG_WITH([fast-mutexes],
-	    AC_HELP_STRING([--with-fast-mutexes], 
+	    AS_HELP_STRING([--with-fast-mutexes], 
 	    [Compile with fast mutexes (default is disabled)]),
 	    [with_fast_mutexes=$withval], [with_fast_mutexes=no])
 
@@ -1745,7 +1745,7 @@ then
 fi
 
 AC_ARG_WITH([atomic-ops],
-	    AC_HELP_STRING([--with-atomic-ops=rwlocks|smp|up],
+	    AS_HELP_STRING([--with-atomic-ops=rwlocks|smp|up],
 	    [Implement atomic operations using pthread rwlocks or atomic CPU
              instructions for multi-processor (default) or uniprocessor
              configuration]), , [with_atomic_ops=smp])
@@ -2537,7 +2537,7 @@ fi
 tools_dirs=""
 
 AC_ARG_WITH([mysqlmanager],
-  AC_HELP_STRING([--with-mysqlmanager], [Build the mysqlmanager binary: yes/no (default: build if server is built.)]),,)
+  AS_HELP_STRING([--with-mysqlmanager], [Build the mysqlmanager binary: yes/no (default: build if server is built.)]),,)
 
 if test "$with_mysqlmanager" = "yes" -o \
         '(' "$with_mysqlmanager:$with_server" = ":yes" -a \
