From 2c016fa2f89bb56799f4b5d74bae13bfe6c06f96 Mon Sep 17 00:00:00 2001
From: esaapriliasalsabila <esaapriliasalsabila@gmail.com>
Date: Wed, 24 Jul 2024 06:35:32 +0800
Subject: [PATCH] add autogen.sh

---
 autogen.sh | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100755 autogen.sh

diff --git a/autogen.sh b/autogen.sh
new file mode 100755
index 0000000..07e632a
--- /dev/null
+++ b/autogen.sh
@@ -0,0 +1,2 @@
+#!/bin/sh
+aclocal && automake --add-missing && autoconf
