From 61f548741216ff90fb89f0cedd0d6a52e6defb9b Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Tue, 23 Jul 2024 06:27:09 +0800
Subject: [PATCH] Add files via upload

---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 94a9d5b..f858497 100644
--- a/configure.in
+++ b/configure.in
@@ -19,7 +19,7 @@ AC_CANONICAL_SYSTEM
 # USTAR format gives us the possibility to store longer path names in
 # TAR files, the path name is split into two parts, a 155 chacater
 # first part and a 100 character second part.
-AM_INIT_AUTOMAKE([1.9 tar-ustar])
+AC_PROG_MAKE_SET
 AC_PROG_LIBTOOL
 
 AC_CONFIG_HEADERS([include/config.h])
