From f621584e103f02728e586599ef2c83fb7317b960 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sat, 20 Jul 2024 10:14:32 +0800
Subject: [PATCH] Add files via upload

---
 configure | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/configure b/configure
index 6c2663c..ac04617 100644
--- a/configure
+++ b/configure
@@ -73573,14 +73573,14 @@ $as_echo "$ext_output" >&6; }
 
 
 
-if test "$PHP_SYSVMSG" != "no"; then
-  ac_fn_c_check_header_mongrel "$LINENO" "sys/msg.h" "ac_cv_header_sys_msg_h" "$ac_includes_default"
-if test "x$ac_cv_header_sys_msg_h" = xyes; then :
-
-else
-  as_fn_error $? "Cannot enable System V IPC support, sys/msg.h is missing" "$LINENO" 5
-
-fi
+#if test "$PHP_SYSVMSG" != "no"; then
+#  ac_fn_c_check_header_mongrel "$LINENO" "sys/msg.h" "ac_cv_header_sys_msg_h" "$ac_includes_default"
+#if test "x$ac_cv_header_sys_msg_h" = xyes; then :
+#
+#else
+#  as_fn_error $? "Cannot enable System V IPC support, sys/msg.h is missing" "$LINENO" 5
+#
+#fi
 
 
 
