--- a/llvm/CMakeLists.txt
+++ b/llvm/CMakeLists.txt
@@ -1145,14 +1145,14 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 include_directories( ${LLVM_INCLUDE_DIR} ${LLVM_MAIN_INCLUDE_DIR})
 
 # when crosscompiling import the executable targets from a file
-if(LLVM_USE_HOST_TOOLS)
-  include(CrossCompile)
-  llvm_create_cross_target(LLVM NATIVE "" Release)
-endif(LLVM_USE_HOST_TOOLS)
-if(LLVM_TARGET_IS_CROSSCOMPILE_HOST)
+#if(LLVM_USE_HOST_TOOLS)
+#  include(CrossCompile)
+#  llvm_create_cross_target(LLVM NATIVE "" Release)
+#endif(LLVM_USE_HOST_TOOLS)
+#if(LLVM_TARGET_IS_CROSSCOMPILE_HOST)
 # Dummy use to avoid CMake Warning: Manually-specified variables were not used
 # (this is a variable that CrossCompile sets on recursive invocations)
-endif()
+#endif()
 
 if( ${CMAKE_SYSTEM_NAME} MATCHES SunOS )
    # special hack for Solaris to handle crazy system sys/regset.h
