From d4cfe8fa0e3b7feac91294472951cab4f07aab9e Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sat, 10 Aug 2024 12:42:00 +0800
Subject: [PATCH] Add files via upload

---
 scripts/jlibtool.c |  4 ++--
 scripts/libtool.mk | 12 ++++++------
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/scripts/jlibtool.c b/scripts/jlibtool.c
index a2606a8..f4023da 100644
--- a/scripts/jlibtool.c
+++ b/scripts/jlibtool.c
@@ -80,7 +80,7 @@ typedef struct {
 
 #ifndef BUILD_RANLIB
 #  if !defined(__EMX__) && !defined(_OSD_POSIX)
-#    define BUILD_RANLIB			"ranlib"
+#    define BUILD_RANLIB			"true"
 #  endif
 #endif
 
@@ -101,7 +101,7 @@ typedef struct {
 #endif
 
 #ifndef TARGET_RANLIB
-#  define TARGET_RANLIB		        "ranlib"
+#  define TARGET_RANLIB		        "true"
 #endif
 
 /*
diff --git a/scripts/libtool.mk b/scripts/libtool.mk
index ad31a7f..93c5230 100644
--- a/scripts/libtool.mk
+++ b/scripts/libtool.mk
@@ -105,14 +105,14 @@ LIBTOOL_VERBOSE=$(if ${VERBOSE},--debug,--silent)
 
 OBJ_EXT = lo
 
-COMPILE.c = ${LIBTOOL} ${LIBTOOL_VERBOSE} --target=${TARGET_SYSTEM} --mode=compile ${TARGET_CC}
-HOST_COMPILE.c = ${LIBTOOL} ${LIBTOOL_VERBOSE} --mode=compile ${BUILD_CC}
+COMPILE.c = ${LIBTOOL} ${LIBTOOL_VERBOSE} --target=${TARGET_SYSTEM} --mode=compile -shared ${TARGET_CC}
+HOST_COMPILE.c = ${LIBTOOL} ${LIBTOOL_VERBOSE} --mode=compile -shared ${BUILD_CC}
 
-LINK.c = ${LIBTOOL} ${LIBTOOL_VERBOSE} --target=${TARGET_SYSTEM} --mode=link ${TARGET_CC}
-HOST_LINK.c = ${LIBTOOL} ${LIBTOOL_VERBOSE} --mode=link ${BUILD_CC}
+LINK.c = ${LIBTOOL} ${LIBTOOL_VERBOSE} --target=${TARGET_SYSTEM} --mode=link -shared ${TARGET_CC}
+HOST_LINK.c = ${LIBTOOL} ${LIBTOOL_VERBOSE} --mode=link -shared ${BUILD_CC}
 
-COMPILE.cxx = ${LIBTOOL} ${LIBTOOL_VERBOSE} --target=${TARGET_SYSTEM} --mode=compile ${CXX}
-LINK.cxx = ${LIBTOOL} ${LIBTOOL_VERBOSE} --target=${TARGET_SYSTEM} --mode=link ${CXX}
+COMPILE.cxx = ${LIBTOOL} ${LIBTOOL_VERBOSE} --target=${TARGET_SYSTEM} --mode=compile -shared ${CXX}
+LINK.cxx = ${LIBTOOL} ${LIBTOOL_VERBOSE} --target=${TARGET_SYSTEM} --mode=link -shared ${CXX}
 
 PROGRAM_INSTALL = ${LIBTOOL} ${LIBTOOL_VERBOSE} --target=${TARGET_SYSTEM} --mode=install ${INSTALL}
 
