From da73982e82730dbbb91dee6b56504d079d1c074d Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sat, 10 Aug 2024 17:19:40 +0800
Subject: [PATCH] Add files via upload

---
 scripts/libtool.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/libtool.mk b/scripts/libtool.mk
index ad31a7f..983d7a5 100644
--- a/scripts/libtool.mk
+++ b/scripts/libtool.mk
@@ -74,8 +74,8 @@ ifeq "${LIBTOOL}" "JLIBTOOL"
     # binary!
     ${JLIBTOOL}: ${top_makedir}/jlibtool.c
 	$(Q)mkdir -p $(dir $@)
-	$(Q)echo CC jlibtool.c
-	$(Q)${BUILD_CC} $< -g3 -o $@ ${JLIBTOOL_DEFS}
+	$(Q)echo HOSTCC jlibtool.c
+	$(Q)${HOSTCC} $< -g3 -o $@ ${JLIBTOOL_DEFS}
 
     jlibtool: ${JLIBTOOL}
 
