From eb0e9aa23aaa749cb2d69c14edb508c14c930c29 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sat, 10 Aug 2024 18:49:41 +0800
Subject: [PATCH] Add files via upload

---
 scripts/libtool.mk | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/scripts/libtool.mk b/scripts/libtool.mk
index ad31a7f..2c087f7 100644
--- a/scripts/libtool.mk
+++ b/scripts/libtool.mk
@@ -38,6 +38,13 @@ ifeq "${LIBTOOL}" "JLIBTOOL"
     # Pass compiler and ranlib paths through to jlibtool if they're
     # defined in the environment.  This lets us define a separate
     # compiler to build the toolchain and
+    
+    ifndef BUILD_CC
+        ifdef CC
+            BUILD_CC := '${HOSTCC}'
+        endif
+    endif
+    
     ifdef BUILD_CC
         JLIBTOOL_DEFS += -DBUILD_CC=\"${BUILD_CC}\" -DHOST_LINK_C=\"${BUILD_CC}\"
     endif
