From fd1aa2f37f140a3f5b742c831bb37990f5f7c8c8 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sun, 11 Aug 2024 19:30:35 +0800
Subject: [PATCH] Update dlopen.c

---
 scripts/build/make/dlopen.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/scripts/build/make/dlopen.c b/scripts/build/make/dlopen.c
index 8b42e8f..d913f15 100644
--- a/scripts/build/make/dlopen.c
+++ b/scripts/build/make/dlopen.c
@@ -32,7 +32,6 @@ RCSID("$Id: f125d9aa2f6de7ee9ad575e2caee3b891b7df64a $")
 #include <errno.h>
 #include <dlfcn.h>
 #include <unistd.h>
-#include <gnumake.h>
 
 #include "log.h"
 
